src/e2e/java/teammates/e2e/cases/e2e/BaseE2ETestCase.java

```diff
@@ -82,6 +82,12 @@ protected static Url createLocalUrl(String testFileName) throws IOException {
      * Logs in a page using admin credentials (i.e. in masquerade mode).
      */
     protected <T extends AppPage> T loginAdminToPage(AppUrl url, Class<T> typeOfPage) {
+        // When not using dev server, Google blocks log in by automation.
+        // To log in, log in manually to teammates in your browser before running e2e tests.
+        // Refer to teammates.e2e.pageobjects.Browser for more information.
```

Comment:
I think it is more appropriate to throw exception with this message (and result in test fail). It is more explicit than a random NPE error where we need to trace down it to this line.

Reply:
I was actually intending to skip the login process when the production server is used and not to cause failure.